deploy/FlapperInit.sol

```diff
@@ -52,30 +53,58 @@ interface DaiJoinLike {
     function dai() external view returns (address);
 }
 
+interface SplitterLike {
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function farm() external view returns (address);
+    function rely(address) external;
+    function file(bytes32, uint256) external;
+    function file(bytes32, address) external;
+}
+
+interface FarmLike {
+    function setRewardsDistribution(address) external;
+    function setRewardsDuration(uint256) external;
+}
+
 struct FlapperUniV2Config {
-    uint256 hop;
     uint256 want;
     address pip;
+    address pair;
+    address daiJoin;
+    address splitter;
+    bytes32 prevChainlogKey;
+    bytes32 chainlogKey;
+}
+
+struct SplitterConfig {
     uint256 hump;
     uint256 bump;
+    uint256 hop;
+    uint256 burn;
+    address farm;
     address daiJoin;
+    bytes32 splitterChainlogKey;
```

Comment:
Shouldn't we also have a prev splitter key?

Reply:
I think if the standard is to write an init function to support the first onboarding spell (as discussed here https://github.com/makerdao/dss-flappers/pull/16#discussion_r1452292706)  then it might be confusing to also include code for cleanup that will only hypothetically be needed at some point in the future.

Reply:
Yeah could be, it feels a bit annoying the other contracts have the prev keys and not here but I do not have a real strong position on this anyway.

Reply:
Resolving this as I mention I do not have a real strong position here.